v2/spanner-to-sourcedb/terraform/samples/spanner-to-sharded-mysql/terraform.tfvars

```diff
@@ -0,0 +1,138 @@
+common_params = {
+  # The project where the resources will be deployed
+  project = "<YOUR_PROJECT_ID>" # Replace with your GCP project ID
+  # The host project in case of Shared VPC setup
+  host_project = "<YOUR_HOST_PROJECT_ID>" # If you are using a shared VPC
+  # The region where the resources will be deployed
+  region = "us-central1"
+  # Optional ID for the migration process
+  migration_id = "migration-123"
+  # Optional name for the replication bucket (defaults to "rr-bucket")
+  replication_bucket = "my-replication-bucket"
```

Comment:
What is this replication_bucket param? Maybe rename it to metadata or dlq based on whatever this is used for?

Reply:
This is the bucket that hosts the following - 

1. DLQ
2. Source shards file
3. Session file

Because it may contain DLQ records (data) and the other two files (metadata), I did not name it as `metadata` and decided to stick with `replication_bucket`. Can change this though if you have a suggestion?

Reply:
Sounds good! No better name suggestions other than adding descriptive comments.